RS3NXTRefactorer.java

```diff
@@ -24,6 +24,9 @@
 public class RS3NXTRefactorer extends GhidraScript {
 
     private static String TODO_DESC = "<RS3 Refactorer: TODO>";
+    
+    //Enable this to print out packet info, copy and paste compatible with OpenRS3
+    private static Boolean printOpenRS3PacketFormat = false;
```

Comment:
For now, I'd always print the packets, and omit the project name in the variable. We could make an options dialog for this later on, or just dumping it straight to a JSON file later on.